.env.example

```
@@ -29,4 +29,10 @@ CORS_ALLOWED_ORIGINS='http://localhost:3000,http://localhost:8000'
 
 SWAGGER_UI_PATH='/api/schema'
 
-ADMIN_EMAILS =  "admin@gmail.com,admin2@gmail.com"
\ No newline at end of file
+ADMIN_EMAILS =  "admin@gmail.com,admin2@gmail.com"
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)